README.md

```diff
@@ -1,16 +1,17 @@
-deoplete.nvim
-=============
+# deoplete.nvim
+> Dark powered asynchronous completion framework for neovim/Vim8
 
 [![Build Status](https://travis-ci.org/Shougo/deoplete.nvim.svg?branch=master)](https://travis-ci.org/Shougo/deoplete.nvim)
-
 [![Join the chat at https://gitter.im/Shougo/deoplete.nvim](https://badges.gitter.im/Shougo/deoplete.nvim.svg)](https://gitter.im/Shougo/deoplete.nvim?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Doc](https://img.shields.io/badge/doc-%3Ah%20deoplete-orange.svg)](doc/deoplete.txt)
 
 Deoplete is the abbreviation of "dark powered neo-completion".  It
-provides an asynchronous keyword completion system in the
-current buffer.
-To view the current options, please consult the
-[documentation](https://github.com/Shougo/deoplete.nvim/blob/master/doc%2Fdeoplete.txt).
+provides an extensible and asynchronous completion framwork for
+neovim/vim8.
```

Comment:
It should be `neovim/Vim8`

Reply:
Yeah, I will Update it  this PR is still WIP, and I am using github web editor.  and when done, I will squash all commits.